db/schema.rb

```diff
@@ -255,8 +266,8 @@
   create_table "provider_users_providers", force: :cascade do |t|
     t.bigint "provider_id", null: false
     t.bigint "provider_user_id", null: false
-    t.datetime "created_at", default: -> { "now()" }, null: false
-    t.datetime "updated_at", default: -> { "now()" }, null: false
+    t.datetime "created_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
```

Comment:
Can we preserve `now()`? Not sure `CURRENT_TIMESTAMP` will do what's expected.

Reply:
I think this is an acknowledged difference in behaviour between postgres versions, and Rails has decided not to care? https://github.com/yahonda/rails/commit/4181b677e5164a18fe25767c5bf9eeb3df800231